src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css

```diff
@@ -22,40 +22,73 @@
   --sst-dark-color: var(--sst-sunpy-logo-gray);
   --sst-darker-color: #333333;
   --sst-darkest-color: #222222;
-  --sst-lightest-color: rgb(251, 251, 251);
-  --sst-lighter-color: rgb(231, 231, 231);
+  --sst-lightest-color: rgb(255, 255, 255);
+  --sst-lighter-color: rgb(245, 245, 245);
   --sst-light-color: rgb(206, 206, 206);
 
-  /* Component Colours */
+  /* Component Colors */
   /* If you add or change the names of these please update the docs */
   --sst-header-background: var(--sst-dark-color);
   --sst-header-text: var(--sst-lighter-color);
   --sst-sidebar-background-color: var(--pst-color-background);
 
-  /* sphinx-design themeing */
+  /* sphinx-design theming */
   --sd-color-card-border-hover: var(--sst-accent-color-bright);
 }
 
 html[data-theme="light"] {
+  /* A lot of these colors we have inherited from pst however, I have copied them all */
+  /* in here to make it obvious which colors are in use, and to shield us from any */
+  /* upstream changes. */
   --pst-color-primary: var(--sst-darker-color);
   --pst-color-secondary: var(--sst-accent-color-muted);
+  --pst-color-accent: var(--sst-sunpy-logo-secondary-color);
+  --pst-color-success: rgb(0, 132, 63);
+  --pst-color-info: rgb(39, 107, 233);
+  --pst-color-warning: rgb(238, 144, 64);
+  --pst-color-warning-text: rgb(255, 255, 255);
+  --pst-color-danger: rgb(215, 45, 71);
+  --pst-color-background: var(--sst-lightest-color);
+  --pst-color-on-background: var(--sst-lightest-color);
+  --pst-color-surface: var(--sst-lighter-color);
+  --pst-color-on-surface: var(--sst-darkest-color);
+  --pst-color-target: rgb(243, 207, 149);
   --pst-color-link: var(--sst-accent-color-muted);
+  --pst-color-inline-code: rgb(145, 37, 131);
+  --pst-color-inline-code-links: rgb(238, 144, 64);
+  --pst-color-text-base: var(--sst-darkest-color);
+  --pst-color-text-muted: var(--sst-darker-color);
+
   --sst-footer-background-color: var(--sst-lightest-color);
 }
 
 html[data-theme="dark"] {
-  /* Swap primary and secondary for dark theme */
-  --pst-color-secondary: var(--sst-darker-color);
+  /* A lot of these colors we have inherited from pst however, I have copied them all */
+  /* in here to make it obvious which colors are in use, and to shield us from any */
+  /* upstream changes. */
   --pst-color-primary: var(--sst-accent-color-bright);
+  --pst-color-secondary: var(--sst-darker-color);
```

Comment:
The jump to top thing seems to be because this colour is too close to the `--pst-color-secondary-text` colour.

Reply:
Also curious why I didn't copy those colours in.

Reply:
Also, also, curious as to why we don't have the background / text combos in the test page I copied out of PST.

Reply:
Want me to copy more colors from the main theme we might have missed?